arviz/tests/test_stats.py

```diff
@@ -104,7 +104,7 @@ def test_waic():
 
         step = pm.Metropolis()
         trace = pm.sample(100, step)
-        calculated_waic = pm.waic(trace)
+        calculated_waic = waic(trace)
```

Comment:
There is no `modelcontext` in ArviZ, thus you should explicitly pass the model to  `waic`